xyz.mx_moment.moment.yaml

```diff
@@ -0,0 +1,783 @@
+id: xyz.mx_moment.moment
+runtime: org.kde.Platform
+sdk: org.kde.Sdk
+runtime-version: '5.15-21.08'
+command: moment
+finish-args:
+- --share=ipc
+- --share=network
+- --socket=x11
+- --socket=wayland
+- --socket=pulseaudio
+- --device=dri
+- --talk-name=org.freedesktop.Notifications
+rename-icon: moment
+rename-desktop-file: moment.desktop
+rename-appdata-file: moment.metainfo.xml
+cleanup:
+- /app/include
+- /app/usr/tests
+- /app/lib/cmake
+- /app/bin/cairosvg
+- /app/bin/chardetect
+- /app/bin/futurize
+- /app/bin/jsonschema
+- /app/bin/pasteurize
+- /app/bin/pwiz.py
+modules:
+- name: pyotherside
+  buildsystem: qmake
+  make-install-args:
+  - INSTALL_ROOT=/app
+  post-install:
+  - mkdir -p /app/lib/qml
+  - ln -s /app/usr/lib/qml/io /app/lib/qml
+  sources:
+  - type: archive
+    url: https://github.com/thp/pyotherside/archive/1.5.9.tar.gz
+    sha256: 189cb0b973e40fcb6b95fd51b0bcd6cc8494b514d49ffe966ec488cf05bbf51e
+- name: olm
+  buildsystem: cmake-ninja
+  sources:
+  - type: git
+    url: https://gitlab.matrix.org/matrix-org/olm.git
+    tag: 3.2.2
+    commit: 3745ea57bbce319ac2f190e02062e45a46d23471
+  disable-shallow-clone: true
+  config-opts:
+  - -DCMAKE_BUILD_TYPE=Release
+- name: libzen
+  subdir: Project/GNU/Library
+  config-opts:
+  - --enable-shared
+  - --disable-static
+  cleanup:
+  - /bin
+  - /include
+  - /lib/pkgconfig
+  - /lib/*.la
+  sources:
+  - type: archive
+    url: https://mediaarea.net/download/source/libzen/0.4.37/libzen_0.4.37.tar.xz
+    sha256: 38c0a68b715b55d6685d2759eecda040adf37bd066955d79a5d01f91977bd9a0
+- name: libmediainfo
+  subdir: Project/GNU/Library
+  config-opts:
+  - --enable-shared
+  - --disable-static
+  - --with-libcurl
+  cleanup:
+  - /bin
+  - /include
+  - /lib/pkgconfig
+  - /lib/*.la
+  sources:
+  - type: archive
+    url: https://mediaarea.net/download/source/libmediainfo/19.09/libmediainfo_19.09.tar.xz
+    sha256: ff06e1a449dfbe6f2c51f27ae1187d3e72386cb54476fbb189ffaacf845f478e
+- name: python3-cffi
```

Comment:
Do these come from the pip generator? Could they be split to separate file?

Reply:
Honestly I'm not sure, but I'd guess so. I've "inherited" most of this manifest from a previous inactive upstream. I could try splitting it out, sure, but I had issues when using flatpak-pip-generator to generate the entire list of dependencies.

Reply:
That's fair, but please look into it when you have spare time.